javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,87 @@
+//FreeCOdeCamp Exercise link
+//https://www.freecodecamp.org/myselfankitaa
+
+//Age-ify (A future age calculator)
+const yearOfBirth = 1988;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
```

Comment:
Try apply template literals (see last review comment and try apply that here)

Reply:
Noted, I made the changes accordingly.

Reply:
Please check and let me know if the code are ok now and I can merge the PR.
Thanks for your valuable suggestions.